apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/about.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <LayoutsDetailPage>
+    <template #header>
+      <PageHeader title="About">
+        <template #prefix>
+          <BreadCrumbs :crumbs="[]" />
+        </template>
+      </PageHeader>
+    </template>
+    <template #side>
+      <SideNavigation
+        :title="collectionEvent?.name"
+        :items="[
+          { id: 'intro', label: 'Introduction' },
+          { id: 'standards', label: 'Standards' },
+          { id: 'contact', label: 'Contact' },
+          { id: 'privacy', label: 'Privacy' },
+          { id: 'terms', label: 'Terms' },
+          { id: 'other', label: 'Links' },
+        ]"
+      />
+    </template>
+    <template #main>
+      <ContentBlocks>
+        <ContentBlock id="intro" title="Introduction">
+          This meta-data catalogue was commissioned by the University Medical
+          Centre Groningen (UMCG). The aim of the UMCG Research Data Catalogue
+          is to increase the visibility of UMCG data and sample resources and
+          thereby stimulate their findability, accessibility and reuse, three
+          major pillars of the
+          <a
+            href="https://www.go-fair.org/fair-principles/"
+            rel="noopener noreferrer"
+            target="_blank"
+            style="background-color: transparent; color: rgb(5, 99, 193)"
+            >FAIR principles</a
+          >. The UMCG Research Data Catalogue includes a wide variety of data
+          and sample resources, from large population-based cohorts and clinical
+          biobanks to studies and everything in between. Some resources date
+          back all the way to 1962.&nbsp; The catalogue was developed by the
+          <a
+            href="https://umcgresearch.org/w/gcc"
+            rel="noopener noreferrer"
+            target="_blank"
```

Comment:
the is a external link img that is used in the cohort detail page that indicates to the user that the link points to a external source